tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -78,11 +86,12 @@ TEST (nnstreamerFilterTvm, openClose00)
 {
   int ret;
   void *data = NULL;
-  gchar *model_file;
+  gchar *model_file, *model_name;
   const gchar *root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
   GstTensorFilterProperties prop;
+  model_name = g_strdup_printf("tvm_add_one_%s.so_", ARCH);
```

Comment:
recommend to add static function to get the model file to reduce duplicated lines :)

Reply:
Thanks! Updated